### ISSUE TITLE
Please can you add the current days Gas m3 usage

### ISSUE THREAD

**Issue by gb095666** [BottlecapDave/HomeAssistant-OctopusEnergy#390]
Currently there is a Gas sensor for accumulative consumption in m3 for the previous period, however there is no corresponding entity for the current accumulative period, is it possible that could be created ?

The following sensor shows consumption in m3 for the previous period:-
sensor.octopus_energy_gas_xxx_yyy_previous_accumulative_consumption

Is it possible to create the corresponding sensor in m3 for the current period, i.e.:-
sensor.octopus_energy_gas_xxx_yyy_accumulative_consumption

**Comment by BottlecapDave** [comment]
Hello.

There is not currently a corresponding sensor because the Home Mini only reports in kwh. The two sensors exist for the previous consumption because originally the energy dashboard only accepted m3 data, but the APIs reported in either m3 or kwh depending on the smart meter. Therefore I reported both so that people got the actual reported value for their meter, and then a potentially estimated one for the energy dashboard.

If I was to add a sensor, it would be estimated like it is in certain scenarios for the previous consumption sensors. Out of curiosity, what is your use case for requiring this sensor?

**Comment by gb095666** [comment]
Hi Dave.

Since I moved over to Octopus I noticed that the IHD doesn't show the actual cost of both Electric and Gas if you are on one of their intelligent rates, because the IHD doesn't know/use the correct rates, which makes the IHD a waste of time to be honest.

Using your integration allows me to pull actual usage and correct tariff rates from my Smart Meter to enable me to create a similar IHD display but with the correct costs calculated etc.

However I have noticed that on the Octopus website the Gas usage is calculated and displayed as m3, so I was hoping the current m3 Gas usage could be shown to allow me to compare whats displayed on the Octopus website.
